tiledb/tests/test_libtiledb.py

```diff
@@ -1160,9 +1153,14 @@ def test_incomplete_dense(self):
         with tiledb.from_numpy(path, data, tile=use_tile) as A:
             pass
 
-        # create context with 1 MB memory budget
-        config = tiledb.Config({'sm.memory_budget': 2 * 1024**2})
+        # create context with 1 MB memory budget (2 MB total, 1 MB usable)
+        config = tiledb.Config({'sm.memory_budget': 2 * 1024**2,
+                                'py.init_buffer_bytes': 1024**2 })
```

Comment:
@stavrospapadopoulos do you foresee any problem with putting a custom value in the Config object, in order to maintain a consistent API? Seems to work... (alternatively, could have a separate `py_config` object)